src/Makefile

```diff
@@ -55,9 +55,11 @@ CFLAGS = \
 
 # Compile flags for linux / osx
 ifeq ($(OS),linux)
+CFLAGS += -fopenmp
 SHOBJ_LDFLAGS ?= -Wl,-Bsymbolic,-Bsymbolic-functions -fopenmp -shared -ldl -lpthread
 else
-SHOBJ_LDFLAGS ?= -bundle -undefined dynamic_lookup -ldl -lpthread -fopenmp
+CFLAGS += -Xpreprocessor -fopenmp
+SHOBJ_LDFLAGS ?= -bundle -undefined dynamic_lookup -ldl -lpthread -Xpreprocessor -fopenmp -lomp
```

Comment:
not sure if we need both `-fopenmp -lomp`
do we need openmp flag on both `CFLAGS` and `SHOBJ_LDFLAGS` ?